android/src/main/java/com/rnappauth/RNAppAuthModule.java

```diff
@@ -498,6 +574,21 @@ private AuthorizationServiceConfiguration createAuthorizationServiceConfiguratio
         );
     }
 
+    private void warmChromeCustomTab(Context context, final String issuer) {
```

Comment:
This is a great idea, but I think this should be optional, so the user would be able to opt out of this and _only_ prefetch the configuration.

Reply:
Agreed, I'll add a boolean to the prefetch method.